feature-libs/checkout/components/order-confirmation/components/guest-register-form/guest-register-form.component.ts

```diff
@@ -30,15 +31,15 @@ export class GuestRegisterFormComponent implements OnDestroy {
   );
 
   constructor(
-    protected userService: UserService,
+    protected userRegisterFacade: UserRegisterFacade,
     protected routingService: RoutingService,
     protected authService: AuthService,
     protected fb: FormBuilder
   ) {}
```

Comment:
Could you add constructor migration for this?

Reply:
Added.